feat: add C implementation for `stats/base/dists/bernoulli/skewness` (#3458)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/base/dists/bernoulli/skewness/README.md

@license Apache-2.0

> Copyright (c) 2024 The Stdlib Authors.

Reviewer: ```suggestion
Copyright (c) 2018 The Stdlib Authors.
```

<!-- /.examples -->

<!-- C interface documentation. -->

* * *

<section class="c">

## C APIs

<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->

<section class="intro">

</section>

<!-- /.intro -->

<!-- C usage documentation. -->

<section class="usage">

### Usage

```c
#include "stdlib/stats/base/dists/bernoulli/skewness.h"
```

> #### stdlib_base_dists_bernoulli_skewness( P )

Reviewer 2: ```suggestion
#### stdlib_base_dists_bernoulli_skewness( p )
```

> Returns the skewness of a Bernoulli distribution for probability 'p'

Reviewer 2: ```suggestion
Returns the skewness of a Bernoulli distribution with success probability 'p'.
```

```c
double y = stdlib_base_dists_bernoulli_skewness( 0.1 );
// returns ~2.667

y = stdlib_base_dists_bernoulli_skewness( 0.5 );
// returns ~0.0
```

The function accepts the following arguments:

> -   **y**: `[in] double` - `y` probability

Reviewer 2: ```suggestion
-   **p**: `[in] double` success probability.
```

```c
double stdlib_base_dists_bernoulli_skewness( const double p );
```

</section>

<!-- /.usage -->

<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->

<section class="notes">

</section>

<!-- /.notes -->

<!-- C API usage examples. -->

<section class="examples">

### Examples

```c
#include "stdlib/stats/base/dists/bernoulli/skewness.h"
#include <stdlib.h>
#include <stdio.h>

int main( void ) {
    double p;
    double y;
    int i;

    for ( i = 0; i < 25; i++ ) {
        p = ( (double)rand() / (double)RAND_MAX ) ;
        y = stdlib_base_dists_bernoulli_skewness( p );

>         printf( "x: %lf , H(X;p): %lf\n", p , y );

Reviewer 2: Why `H` instead of `skew`, as in the JS example?
